examples/cluster/plot_optics.py

```diff
@@ -0,0 +1,146 @@
+"""
+===================================
+Demo of OPTICS clustering algorithm
+===================================
+
+Finds core samples of high density and expands clusters from them.
+"""
+
+# Authors: Shane Grigsby (2016), Amy X. Zhang (2012)
```

Comment:
Could you replicate the style of Authors as found in all other files? (Which is mostly one author per line...)

Also the year need not be included. If preferred, emails may be added